### PR TITLE
Removed validation endpoint. Closes #647

### LINKED ISSUE #647: Remove Validation Endpoint
Remove the validation endpoint.

### PR DESCRIPTION
**Pull request by kshakir**
I believe I got all the dead code out, but would appreciate reviewers eyes on this.

### CODE REVIEW

File: engine/src/main/scala/cromwell/webservice/CromwellApiService.scala [hunk 1]
```diff
@@ -25,14 +25,14 @@ trait SwaggerService extends SwaggerUiResourceHttpService {
...
-    Props(classOf[CromwellApiServiceActor], workflowManagerActorRef, validateActorRef, config)
+  def props(workflowManagerActorRef: ActorRef, config: Config): Props = {
+    Props(classOf[CromwellApiServiceActor], workflowManagerActorRef, config)
   }
 
   def traceName(name: String) = Monitor.traceName(name)
 }
 
-class CromwellApiServiceActor(val workflowManager: ActorRef, val workflowDescriptorMaterializer: ActorRef, config: Config)
+class CromwellApiServiceActor(val workflowManager: ActorRef, config: Config)
```
Comment (kshakir) [inline engine/src/main/scala/cromwell/webservice/CromwellApiService.scala]: NOTE: I almost missed removing this cruft, and why I prefer `override val` to just `val`, even if it's ugly. 😉

**Comment by kshakir** [comment]
Wheel officially says @mcovarr and @scottfrazer. But in my biased opinion this PR isn't that big, and anyone can review.

**Comment by geoffjentry** [comment]
LGTM

**Comment by kshakir** [comment]
Switching reviewers from ~~@scottfrazer~~ to tribute @kcibul.

**Comment by kcibul** [comment]
👍 

Also built/ran serer and verified that it doesn't show up in the swagger

**Comment by kcibul** [comment]
re 👍  so pullapprove might pick it up

**Comment by kshakir** [comment]
PullApprove be damned! I'm merging! :steam_locomotive::train::train::train::dash:

**Comment by geoffjentry** [comment]
You rebel

**Pull request merged**